generators/chipyard/src/main/scala/DigitalTop.scala

```diff
@@ -33,7 +33,6 @@ class DigitalTop(implicit p: Parameters) extends ChipyardSystem
 class DigitalTopModule[+L <: DigitalTop](l: L) extends ChipyardSystemModule(l)
   with testchipip.CanHaveTraceIOModuleImp
   with testchipip.CanHavePeripheryBlockDeviceModuleImp
-  with testchipip.CanHavePeripherySerialModuleImp
```

Comment:
Why did this get removed?

Reply:
`CanHavePeripherySerial` exists only as a lazy trait now, I removed the corresponding `ModuleImp` trait.